niap-cc/Permissions/TransactIds/app/src/main/java/com/android/certifications/niap/permissions/transactids/ProxyChecker.java

```diff
@@ -40,12 +40,14 @@ public class ProxyChecker {
         alt_descriptor.put("android.graphicsenv.IGpuService","GPU_DESCRIPTOR");
         alt_descriptor.put(WINDOW_DESCRIPTOR,"WINDOW_DESCRIPTOR");
         alt_descriptor.put(EUICC_CONTROLLER_DESCRIPTOR,"EUICC_CONTROLLER_DESCRIPTOR");
+        alt_descriptor.put(PDB_DESCRIPTOR,"PDB_DESCRIPTOR");
+        alt_descriptor.put(SYSTEM_CONFIG_DESCRIPTOR,"SYSTEM_CONFIG_DESCRIPTOR");
+
         //alt_descriptor.put("com.android.internal.widget.ILockSettings","LOCK_SETTINGS_DESCRIPTOR");
         for(String k:alt_descriptor.keySet()){
-            System.out.println(
-                    String.format("public static final String %s = \"%s\";",
-                            alt_descriptor.get(k),
-                            k));
+            System.out.printf("public static final String %s = \"%s\";%n",
```

Comment:
Here and the `println` statements below; should these be logcat entries instead to make it easier to identify them by their tag?

Reply:
Okay. Modified to use Log.d instead of println.